addons/core/functions/fnc_getSwSettings.sqf

```diff
@@ -65,6 +65,20 @@ if (isNil "_rc") then {
     _value set [TFAR_CODE_OFFSET, _rc];
 };
 
+private _hdo = _value select HALFDUPLEX_OVERRIDE_OFFSET;
```

Comment:
`_hdo` ? 
Readability doesn't hurt. Also I'd like this code to look as similear as possible to the LR code.
so `param` with default value.
And below you use isNumber inst.... Wait. Yeah no. Don't use direct config Access. Use our getConfigValue or what it's called. Our function provides integrated caching.